src/app/components/layout/navbar/navbar.component.ts

```diff
@@ -340,6 +348,7 @@ export class NavbarComponent implements OnInit, AfterViewInit, OnDestroy {
     this.stop();
     this.timer = setInterval(function () {
       this.pingService();
+      this.getServiceStatus(false);
```

Comment:
We can not bind the getServiceStatus call in this auto refresh interval. By doing this  services API will always be running like ping based on the auto refresh timer. The service status call should be only on additional service page and there auto refresh should work.